modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,80 @@
+* Translations added for Help text displayed at *Help > Help > Keyboard Navigation*.
+* For word count purposes these characters are now considered punctuation marks: *$*, *~*, *+*, *|*, *№*, and *`*. They no longer increase a document’s word count.
+* Updated the `codesample` plugin dialog and the `template` plugin dialog to use the `listbox` component to match other dialogs.
+* If the selection contains more than one table cell, Quickbar toolbars are now positioned in the middle of the selection horizontally.
+* Exposed `dataTransfer` property of drag and drop events for elements with a `contenteditable="false"` attribute.
+
+=== Changed
+* The `caption`, `address` and `dt` elements no longer incorrectly allow non-inline child elements when the editor schema is set to _HTML 4_.
+* SVG icons for back and foreground colors now use `class` instead of `id` to identify SVG elements that should change color.
+* Anchor tag elements — `<a>` — no longer incorrectly allow non-inline child elements when the editor schema is set to _HTML 4_.
+
+=== Fixed
+* Right-clicking on a merge tag instance presented different highlighting depending on the host browser.
+* When macOS was the host operating system, pressing *Command+backspace* did not add an undo level.
+* *Ctrl+backspace* and *Ctrl+delete* did not restore the correct insertion point position after a redo operation.
+* In the `tree` component, a selected item in a directory would not stay selected after collapsing the directory.
+* Enabling or Disabling checkboxes would not set the correct classes and attributes.
+* Entering a newline would, in some setups, place the insertion point in the wrong paragraph.
+* Redial would, in some situations, cause select elements not to have an initial value selected when they should have.
+* The Table toolbar was visible even if the table was within a host element with a `contenteditable="false"` attribute set.
+* Quickbar toolbars were incorrectly shown for elements with a `contenteditable="false"` attribute set in a root with a `contenteditable="false"` attribute set.
+* When Chrome was the host browser, adding a newline when the insertion point was placed after a table could, in some specific situations, not generate the expected newline.
+* Show the calculated height and width of Media Embed elements in the `media` plugin dialog.
+* Removing an image that failed to upload from an empty paragraph left the paragraph without a padding `<br>` tag.
+* Allow a Media Embed element to be correctly resized when using the `media` plugin dialog by converting the Media Embed element to a standalone iframe.
+* In some circumstances, an inline alert in the _Search and Replace_ dialog persisted when it was not necessary.
+* Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
+* In Safari running on iOS, Korean characters merged onto the previous line upon typing after inserting a newline by pressing Enter.
```

Comment:
```suggestion
* On iOS Safari, Korean characters will no longer merge onto the previous line after inserting a newline by pressing Enter.
```

Reply:
Incoming change from https://github.com/tinymce/tinymce-docs/pull/2748#discussion_r1211041664 @mkzhx

Reply:
This won't be necessary as it was adapted from the latest version of my changelog entry already.

Reply:
Edits made upstream.